netci.groovy

```diff
@@ -49,168 +49,68 @@ build.cmd /no-node-reuse /no-deploy-extension /${configuration.toLowerCase()}
-SET VSSDK150Install=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\v15.0\\VSSDK\\
-SET VSSDKInstall=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\v15.0\\VSSDK\\
+        def newVsiJob = job(newVsiJobName) {
+            // This opens the set of build steps that will be run.
+            steps {
+                // Indicates that a batch script should be run with the build string (see above)
+                // Also available is:
+                // shell (for unix scripting)
+                batchFile("""
+echo *** Installing 1.0 CLI ***
 
-build.cmd /no-node-reuse /release /skiptests
-""")
+@powershell -NoProfile -ExecutionPolicy Bypass -Command "((New-Object System.Net.WebClient).DownloadFile('https://download.microsoft.com/download/B/9/F/B9F1AF57-C14A-4670-9973-CDF47209B5BF/dotnet-dev-win-x64.1.0.4.exe', 'dotnet-dev-win-x64.1.0.4.exe'))"
+dotnet-dev-win-x64.1.0.4.exe /install /quiet /norestart /log cli_install.log
 
-            // Patch all the MSBuild xaml and targets files from the current roslyn-project-system commit into VS install.
-            batchFile("""
 echo *** Patch the MSBuild xaml and targets ***
-SET VS_MSBUILD_MANAGED=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\Managed
-
+SET VS_MSBUILD_MANAGED=%ProgramFiles(x86)%\\Microsoft Visual Studio\\Preview\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\Managed
 mkdir backup
+build.cmd /no-node-reuse /skiptests /${configuration.toLowerCase()}
 xcopy /SIY "%VS_MSBUILD_MANAGED%" .\\backup\\Managed
-
 xcopy /SIY .\\src\\Targets\\*.targets "%VS_MSBUILD_MANAGED%"
-xcopy /SIY .\\bin\\Release\\Rules\\*.xaml "%VS_MSBUILD_MANAGED%"
-""")
-
-            // Restore roslyn nuget packages
-            batchFile("""
-echo *** Restore Roslyn ***
-pushd %WORKSPACE%\\roslyn
-Restore.cmd
-""")
-
-            // Build the SDK and install .NET Core Templates.
-            batchFile("""
-echo *** Build the SDK and install .NET Core Templates  ***
-SET VS150COMNTOOLS=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\Common7\\Tools\\
-SET DeveloperCommandPrompt=%VS150COMNTOOLS%\\VsMSBuildCmd.bat
-
-echo  *** Call VsMSBuildCmd.bat
-call "%DeveloperCommandPrompt%" || goto :BuildFailed "VsMSBuildCmd.bat"
-
-SET VSSDK150Install=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\v15.0\\VSSDK\\
-SET VSSDKInstall=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\v15.0\\VSSDK\\
-SET MSBUILDEXE=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\15.0\\Bin\\MSBuild.exe
-
-pushd %WORKSPACE%\\sdk
-echo *** Build SDK
-call build.cmd -Configuration release -SkipTests || goto :BuildFailed "SDK"
+xcopy /SIY .\\bin\\${configuration}\\Rules\\*.xaml "%VS_MSBUILD_MANAGED%"
```

Comment:
Instead of doing the xcopy can we just set the env variables for the targets like we do in launchSettings.json?